### PR TITLE
Add preview server stack

### PR DESCRIPTION
**Pull request by enys**

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,5 +1,6 @@
 RUN_ENV=dev
 WORDPRESS_VERSION=5.7.2
+NUXTJS_VERSION=0.0.1
```
Comment (backnight) [inline .env.example:3]: Sure about this default version?

**Review by backnight — COMMENTED** [review]
Looks good to me, except for the NUXTJS version, I think we should set it by default to the one we use

File: .env.example (hunk 1 above)
Comment (enys) [reply .env.example:3]: This more a naming issue. It is not the NUXTJS version ( ie from the nuxtjs dependency) but the version of the project https://github.com/Linkurious/website-nuxtjs/blob/develop/package.json#L3

Comment (backnight) [reply .env.example:3]: Alright then I would use something like "APP_VERSION"

Comment (backnight) [reply .env.example:3]: or NUXTJS_APP_VERSION

Comment (enys) [reply .env.example:3]: what about match the docker image name (and package name) : WEBSITE_NUXTJS_VERSION

Comment (backnight) [reply .env.example:3]: go for it

**Pull request merged**